google/cloud/bigquery/job/base.py

```diff
@@ -171,6 +171,33 @@ def __setattr__(self, name, value):
             )
         super(_JobConfig, self).__setattr__(name, value)
 
+    @property
+    def job_timeout_ms(self):
+        """Optional parameter. Job timeout in milliseconds. If this time limit is exceeded, BigQuery might attempt to stop the job.
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfiguration.FIELDS.job_timeout_ms
+        e.g.
+
+            job_config = bigquery.QueryJobConfig( jobtimeout = 5000 )
+            or
+            job_config.jobtimeout = 5000
+
+        Raises:
+            ValueError: If ``value`` type is invalid.
+        """
+
+        # None as this is an optional parameter.
+        if self._properties.get("jobTimeoutMs"):
```

Comment:
Let's use the `_int_or_none` helper for this instead: https://github.com/googleapis/python-bigquery/blob/53aad826a55dd1c36d014c80ada515273e8aa92a/google/cloud/bigquery/_helpers.py#L860

BigQuery often serializes integers as strings in the JSON response.

Reply:
I will see about using that. Thanks for the tip.